packages/react/src/components/Tabs/next/Tabs.js

```diff
@@ -241,19 +459,39 @@ TabList.propTypes = {
    * Specify an optional className to be added to the container node
    */
   className: PropTypes.string,
+
   /**
    * Specify whether component is contained type
    */
-
   contained: PropTypes.bool,
+
   /**
    * If using `IconTab`, specify the size of the icon being used.
    */
   iconSize: PropTypes.oneOf(['default', 'lg']),
+
+  /**
+   * Provide the props that describe the left overflow button
+   */
+  leftOverflowButtonProps: PropTypes.object,
```

Comment:
Random question, are these props that we're porting over from the existing implementation? 👀

Reply:
Yep! Exactly